### ISSUE TITLE
Cannot read property 'name' of undefined

### ISSUE THREAD

**Issue by mohamednizar** [shuyu/angular-material-fileinput#76]
I'm trying to upload multiple images.I use to bind available remote files on the edit form.See the code 
```
$timeout(
      function () {
                    var count = 0;
                    angular.forEach(angular.copy(Service.images), function (element) {
                        count += 1;
                        vm.api.addRemoteFile(element, 'Pic '+count, 'image');
                    });
                }
      ,vm.api)
```

Here my input container.

 ```
<md-input-container class="md-icon-float md-block">
    <lf-ng-md-file-input lf-files='vm.picture' lf-browse-label='Select Images' lf-maxcount="5" ng change="vm.updateImage()" lf-api="vm.api" lf-mimetype="image/*" preview drag multiple progress> </lf-ng-md-file-input>
    <md-progress-linear md-mode="indeterminate"  ng-show="vm.isProgress"></md-progress-linear>
    <input name="pictureName" ng-model='vm.service.images' required type="hidden">
</md-input-container>

```

When I'm editing the form this error returning.
```
TypeError: Cannot read property 'name' of undefined
    at http://myrejuvenate.localhost:3000/bower_components/lf-ng-md-file-input/dist/lf-ng-md-file-input.js:686:17
    at Array.every (native)
    at http://myrejuvenate.localhost:3000/bower_components/lf-ng-md-file-input/dist/lf-ng-md-file-input.js:684:39
    at processQueue (http://myrejuvenate.localhost:3000/bower_components/angular/angular.js:16385:28)
    at http://myrejuvenate.localhost:3000/bower_components/angular/angular.js:16401:27
    at Scope.$eval (http://myrejuvenate.localhost:3000/bower_components/angular/angular.js:17684:28)
    at Scope.$digest (http://myrejuvenate.localhost:3000/bower_components/angular/angular.js:17497:31)
    at http://myrejuvenate.localhost:3000/bower_components/angular/angular.js:17723:26
    at completeOutstandingRequest (http://myrejuvenate.localhost:3000/bower_components/angular/angular.js:5966:10)
    at http://myrejuvenate.localhost:3000/bower_components/angular/angular.js:6245:7
```
Wha is the issue with this?

**Comment by mohamednizar** [comment]
This was fixed with #76

**Comment by nadeesha1111** [comment]
change the 686 line of  lf-ng-md-file-input.js as 	'if(lfFile==undefined?'':lfFile.name == file.name) {'
